### DISCUSSION TITLE
iSpindel AP connectio

### DISCUSSION THREAD

**Discussion by bazantews** [universam1/iSpindel#624] [Q&A] [answered]
Hello guys, i setup my iSpindel successfully, but when i enter in config mode, it says "not connected to network" any solution for this trouble?![image](https://github.com/user-attachments/assets/cf61e7cc-28b4-4421-98cf-1015d6fe5949)

**Comment by fmjensen** [comment, marked as answer]
This is normal AFAIK. 
When iSpindle boot it will connect to "brewpiless" wifi every time it wakes up to send data...

**Reply by bazantews** [reply]
Ohh, i dosn't know it's normal. I think its an problem of my wemos, tks. Btw, u know how i can connect my iSpindel to brewpiless? Like to brewpiless receive the data?

**Reply by fmjensen** [reply]
We will need a wee bit more context on what you waht to achieve with your iSpindle to give advice :-)

I my self have my iSpindle setup to use MQTT to send data to my Home Assistant server where I process it using Node Red so I can see the data in graphs from the database historic data

**Comment by wilbrod** [comment]
I'd suggest you look at videos on YouTube on how the ispindel works.

When you're in configuration mode, the unit isn't transmitting any
information. Only once you're out of configuration mode will the unit start
its cycle of sleep/wake-up.

The ispindel will only connect to the server after wake-up for a brief
moment, only to send data and will go back to sleep for the next cycle.

Of you are connected to it, it cannot connect to your server. If it's in
operation mode, you can't connect to it either. It's one or the other.

**Comment by bazantews** [comment]
As @fmjensen said, its normal. I connected successfully to my brewpiless now. But i need to calibrate the iSpindel, after getting the formula calibration i need to set that formula in configuration page of iSpindel?

**Comment by wilbrod** [comment]
If you want to make changes, you have to reboot in configuration mode. Hit
the reset button on the wemos a few times in a row while the ispindel is
turned on. Depending on your build, there may be pins available you can
short out that would mimic hitting the reset button while not having to
slide the unit out of its cylinder.

On the other hand, additional calibration isn't necessary iny opinion,
built-in formula is good enough.
